### DISCUSSION TITLE
Show which graphs in a dashboard don't support the global filter(s)

### DISCUSSION THREAD

**Discussion by rohitagarwal003** [grafana/grafana#70399] [Dashboards, Panels & Visualizations]
Currently, there's no UI hint that tells when some graphs in a dashboard don't support a global filter. It's a bit confusing to apply a filter and not know which graphs have been updated based on the filter and which haven't.

It would be nice if we show some UI hint that tells which graphs are ignoring the global filter. In an internal system that I used previously, this was shown by adding some strikethrough text at the top of the graphs (e.g. ~variable=value~) that ignored the filter.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82661. Feel free to continue the discussion around this there. Thank you!
